EventsExpress.Core/Services/ContactAdminService.cs

```diff
@@ -77,10 +77,13 @@ public IEnumerable<ContactAdminDto> GetAll(ContactAdminFilterViewModel model, ou
                 : contactAdminMessages;
 
             count = contactAdminMessages.Count();
-            var result = contactAdminMessages
-                .OrderBy(x => x.DateCreated)
-                .Skip((model.Page - 1) * model.PageSize)
-                .Take(model.PageSize).ToList();
+            var result = model.Page == 0
```

Comment:
We would not want to do these ternary operators. It gives a user possibility to break our system, if there are too many items. I would suggest just return no items, if it is page 0, since that is what he asked for explicitly. If the user did not specify the parameters - we will default and that is what we are trying to fix.